apps/renterd/contexts/config/transform.spec.ts

```diff
@@ -526,29 +518,26 @@ describe('tansforms', () => {
 
 function buildAllResponses() {
   return {
-    autopilotInfo: {
-      state: {
-        id: 'autopilot',
-        configured: true,
-        migrating: true,
-        migratingLastStart: new Date().toISOString(),
-        scanning: true,
-        scanningLastStart: new Date().toISOString(),
-        synced: true,
-        uptimeMS: '333',
-        network: 'mainnet' as const,
-        version: '0.0.0',
-        commit: 'commit',
-        OS: 'os',
-        buildTime: new Date().getTime(),
-        explorer: {
-          enabled: true,
-          url: 'https://api.siascan.com',
-        },
-        startTime: new Date().getTime(),
+    autopilotState: {
+      id: 'autopilot',
+      configured: true,
+      migrating: true,
+      migratingLastStart: new Date().toISOString(),
+      scanning: true,
+      scanningLastStart: new Date().toISOString(),
+      synced: true,
+      uptimeMS: '333',
+      network: 'mainnet' as const,
+      version: '0.0.0',
+      commit: 'commit',
+      OS: 'os',
+      buildTime: new Date().getTime(),
+      explorer: {
```

Comment:
I was surprised to see this in here. Do you extend our state response with this explorer config?

Reply:
Ah great catch, the bus and autopilot state type definitions look a bit off, I will fix that.